orchagent/routeorch.h

```diff
@@ -152,6 +152,10 @@ class RouteOrch : public Orch, public Subject
     bool createFineGrainedNextHopGroup(sai_object_id_t &next_hop_group_id, vector<sai_attribute_t> &nhg_attrs);
     bool removeFineGrainedNextHopGroup(sai_object_id_t &next_hop_group_id);
 
+    void addLinkLocalRouteToMe(sai_object_id_t vrf_id, IpPrefix linklocal_prefix);
+    void delLinkLocalRouteToMe(sai_object_id_t vrf_id, IpPrefix linklocal_prefix);
+    std::string getLinkLocalEui64Addr(void);
```

Comment:
Why is `getLinkLocalEui64Addr `made public. Please keep it private

Reply:
This function call is needed in vrforch.cpp file changes, so made it Public.